### PR TITLE
Refs #313 - Fix tests to match plone.dexterity dynamic schema name

### LINKED ISSUE #313: Update tests to match plone.dexterity changes
See https://github.com/plone/plone.dexterity/pull/137

### PR DESCRIPTION
**Pull request by avoinea**
Fixes #313

### CODE REVIEW

**Comment by avoinea** [comment]
@jenkins-plone-org please run jobs

File: plone/app/dexterity/tests/editing.rst [hunk 1]
```diff
@@ -129,6 +129,7 @@ and saved::
   ...     browser.getControl('Field type').getControl(
   ...         value=factory.title).selected = True
   ...     browser.getControl('Add').click()
+  ...     schema = plonista.lookupSchema()
```
Comment (ale-rt) [inline plone/app/dexterity/tests/editing.rst:132]: Looks good to me, probably `>>> schema = plonista.lookupSchema()` in line 119 can be removed:
https://github.com/plone/plone.app.dexterity/pull/314/files#diff-479ccfb98b9b64526753674cb4ee2672R119

**Review by ale-rt — APPROVED** [review]
LGTM, I just added a clean up suggestion

**Review by ale-rt — APPROVED** [review]

**Comment by ale-rt** [comment]
@jenkins-plone-org please run jobs

**Pull request merged**